.circleci/config.yml

```diff
@@ -637,8 +637,6 @@ jobs:
           echo "(cat docker_commands.sh | docker exec -u jenkins -i "$id" bash) 2>&1" > command.sh
           unbuffer bash command.sh | ts
 
-          echo "Retrieving test reports"
-          docker cp $id:/var/lib/jenkins/workspace/test/test-reports ./ || echo 'No test reports found!'
           if [[ ${BUILD_ENVIRONMENT} == *"coverage"* ]]; then
               echo "Retrieving C++ coverage report"
```

Comment:
should we also move this to a separate step?

Reply:
Isn't it currently in the Run Tests step? I this maybe moving it to the Report Results step is reasonable?